packages/legacy-workbench/src/js/angular/guides/steps/core/plugin.js

```diff
@@ -103,6 +110,51 @@ PluginRegistry.add('guide.step', [
             return stepDescription;
         }
     },
+    {
+        guideBlockName: 'copy-text-element',
+        getSteps: (options, services) => {
+            const $translate = services.$translate;
+            const GuideUtils = services.GuideUtils;
+
+            const text = options.text;
+            const code = document.createElement('code');
+            code.innerText = text;
+
+            const copy = document.createElement('button');
+            const copyToEditorButtonClass = 'guide-copy-to-editor-query-button';
+            copy.className = `btn btn-sm btn-secondary ${copyToEditorButtonClass}`;
+            copy.innerText = $translate.instant('guide.step_plugin.execute-sparql-query.copy-to-editor.button');
```

Comment:
'guide.step_plugin.execute-sparql-query.copy-to-editor.button' is translated as "Copy to editor", but I think it should just be "Copy" or "Copy text", with support for overriding when necessary

Reply:
Changed to `Copy to input` as we discussed the behaviour.

I don't think this should be exposed for now.